app/.gitignore

```diff
@@ -13,3 +13,4 @@ composer.lock
 Homestead.json
 Homestead.yaml
 .env
+/public/fonts
```

Comment:
This change probably isn't needed if you get latest on your master branch and rebase.
Specifically this commit is what you'd want: https://github.com/hhaccessibility/hhaccessibility.github.io/commit/62885f6e9a6281ea8bc25af70745571e3750aa2a

Your master in your fork(https://github.com/passionInfinite/hhaccessibility.github.io) is behind master in the main repository(https://github.com/hhaccessibility/hhaccessibility.github.io/) by about 50 commits which included a change to .gitignore.  

https://github.com/hhaccessibility/hhaccessibility.github.io/wiki/git-commands has some command sequences that could help if you have some trouble.

Reply:
Yea! I synced my branches merges this time. From Next time I will take care of it :+1:

Reply:
ok.  I'll merge this and then merge a commit that removes this line so it isn't forgotten.  It is a pretty small detail but I just don't want to waste the discovery by forgetting about it.